### DISCUSSION TITLE
UI Formatting Issue with failed workflows

### DISCUSSION THREAD

**Discussion by devstewart** [argoproj/argo-workflows#11935] [Q&A] [answered]
Unsure if this is a known issue for not or if there's something wrong on my side, but this is what I'm seeing after upgrade to v3.4.11

![Screenshot 2023-10-03 102237](https://github.com/argoproj/argo-workflows/assets/86795029/e57d1769-8ebc-490c-8c07-3db0f552b30e)

**Comment by terrytangyuan** [comment]
@agilgur5 I remember you or someone else had a PR that fixes this. Could you take a look to see if we need to cherry-pick any commits to v3.4?

**Reply by agilgur5** [reply, marked as answer]
Yep, PR fix out already for this: https://github.com/argoproj/argo-workflows/pull/11908 (already approved as well, just failing on a test flake right now)
